### ISSUE TITLE
Error: Invalid GBK character "\xE5"

### ISSUE THREAD

**Issue by zhouytao** [oklai/koala#488]
When using the Koala compile sass file, if the file appear in Chinese characters, the code will compile error, an error log is below:

```
/*
Error: Invalid GBK character "\xE5"
        on line 2 of D:/workspace/test/public/stylesheets/base/_style.scss
        from line 7 of D:\workspace\test\public\stylesheets\main.scss

1: /**
2:  * 导航
3:  *\/
4: .nav {
5: 	padding: 0;
6: 	margin: 0;
7: 	list-style: none;
```

I see why through forums should be Koala comes sass procedural issues. Sass need to modify the source code or replace version
My solution is:
Koala installation directory into **Koala\rubygems\gems\sass-3.4.9\lib\sass**
Modify **engine.rb** file
In the **require** the bottom add the following line of code to solve the problem of compiler error
`Encoding.default_external = Encoding.find ( 'utf-8')`

**Comment by pageye123** [comment]
yeah,it's right to solve the problem,I also do this when I meet the same  question.But unforturnately,the source code is without the

> Encoding.default_external = Encoding.find ('utf-8')
